components/dataentry/dataentry-controller.js

```diff
@@ -1356,24 +1358,37 @@ trackerCapture.controller('DataEntryController',
         });
         
         $scope.setDisplayTypeForStage($scope.currentStage);
-        
+        //Change this to use flag to determin wich mode to use.
+        $scope.currentStage.timelineDataEntryMode = $scope.timelineDataEntryModes.COMPAREALLDATAENTRYFORM;

```

Comment:
**NOTE:** This needs to be changed to take in a flag. It is currently _hard coded_ to always display compare form.

Reply:
Has been fixed in latest commit.